src/nanoarrow/buffer_inline.h

```diff
@@ -15,14 +15,20 @@
 // specific language governing permissions and limitations
 // under the License.
 
+#ifndef NANOARROW_BUFFER_INLINE_H_INCLUDED
+#define NANOARROW_BUFFER_INLINE_H_INCLUDED
+
 #include <errno.h>
-#include <stddef.h>
-#include <stdlib.h>
+#include <stdint.h>
 #include <string.h>
 
-#include "nanoarrow.h"
+#include "typedefs_inline.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
 
-static int64_t ArrowGrowByFactor(int64_t current_capacity, int64_t new_capacity) {
+static inline int64_t _ArrowGrowByFactor(int64_t current_capacity, int64_t new_capacity) {
```

Comment:
It's not allowed to start names with an underscore: https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html

Reply:
It's used in many C projects though, so most probably can be considered safe.

Reply:
Ah, interesting. I agree it's probably safe. It wouldn't be an issue if we have to change it later for some reason.